Documentation~/en/datachannel.md

```diff
@@ -0,0 +1,56 @@
+# Data Channel

```

Comment:
This is under en, so should be written in English ?

Reply:
Yes, I will request Chris translating it in English